.github/workflows/required-reviewer-approvals.yml

```
@@ -0,0 +1,36 @@
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+      - review_requested
+      - review_request_removed
+      - labeled
+      - unlabeled
+    branches: [main, develop]
+  pull_request_review:
+    types: [submitted, dismissed]
+    branches: [main, develop]
```

actionlint (1.7.12)

[error] "branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)